test/blas.cpp

```diff
@@ -383,6 +384,48 @@ float h_gold_batch[18] = {30.f, 84.f, 138.f,
                           81.f, 36.f, 87.f,
                           69.f, 30.f, 69.f};
 
+#ifndef AF_CPU
+TEST(MatrixMultiply, half) {
+  SUPPORTED_TYPE_CHECK(af_half);
+
+  array A32 = array(3, 3, f32);
+  A32.write<float>((float*)h_lhs, sizeof(float) * 9);
```

Comment:
Prefer constructor instead of calling write.
```suggestion
  array A32(3, 3, h_lhs);
```

Reply:
done